docs/book/get-started.md

```diff
@@ -113,14 +113,25 @@ false
 In addition to running queries, the REPL also lets you define rules:
 
 ```ruby
-> p[x] { a = [1,2,3,4]; a[x] }
-> p[x] > 1
-+---+
-| x |
-+---+
-| 2 |
-| 3 |
-+---+
+> items = ["pizza", "apples", "bread", "coffee"]
+> users = {"bob": {"likes": [0, 2]}, "alice": {"likes": [1, 2, 3]}}
+> likes[[name, item]] { index = users[name].likes[_]; item = items[index] }
+> likes[["alice", item]] # what does alice like?
```

Comment:
I think it would be clearer if we split the first three lines that define constants and rules (116-118) into a separate code block from the queries (119-onward).

After the constants and rules I would include a note:

"The `likes` rule above defines a set of tuples specifying what each user likes."

Reply:
thanks, took care of it